### PR TITLE
Release Notes 4/28/15

### PR DESCRIPTION
**Pull request by mhoeber**
@srpearce  @jimabramson @explorerleslie 

Not much this week.

### CODE REVIEW

File: en_us/release_notes/source/documentation/index.rst [hunk 1]
```diff
@@ -4,6 +4,26 @@ edX Documentation
...
+  change.
+
+* Added the `Exploring Your Dashboard Settings and Profile <Exploring Your
+  Dashboard Settings and Profile Course Staff>`_ topic with profile and account
+  settings information.
+
+==================================
+edX Platform APIs
+==================================
+
```
Comment (mhoeber) [inline en_us/release_notes/source/documentation/index.rst:24]: this will be added shortly

File: en_us/release_notes/source/lms/index.rst [hunk 2]
```diff
@@ -4,6 +4,18 @@ edX Learning Management System
 
 
 ******************
+April 28, 2015
+******************
+
+* Several updates were made in the LMS to support right-to-left languages. In
+  right-to-left languages, the header on the **Course Info page** in now right-
```
Comment (srpearce) [inline en_us/release_notes/source/lms/index.rst]: in --> is

**Comment by explorerleslie** [comment]
:+1:

**Comment by srpearce** [comment]
Announcement draft: http://partners.edx.org/announcements/edx-release-april-28-2015

**Comment by mhoeber** [comment]
@srpearce  :+1:  on announcement

to deal with a small bug, can you publish this from the open edx version of page:  http://open.edx.org/announcements/edx-release-april-28-2015

**Comment by srpearce** [comment]
Sure - will publish the announcement from Open edX.

File: en_us/release_notes/source/links.rst [hunk 3]
```diff
@@ -452,4 +452,6 @@
 
 .. _edX Style Guidelines: http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/style_guides/index.html
 
-.. _Recommender: http://edx.readthedocs.org/projects/edx-partner-course-staff/en/latest/exercises_tools/recommenderXBlock.html#recommenderxblock
```
Comment (jimabramson) [inline en_us/release_notes/source/links.rst:455]: i assume this bit was intentionally removed.

**Pull request merged**